lib/commands/init.js

```diff
@@ -107,17 +107,17 @@ module.exports = Command.extend({
         }
       })
       .then(function() {
-        if (!commandOptions.skipNpm) {
-          return npmInstall.run({
-            verbose: commandOptions.verbose,
-            optional: false
+        if (!commandOptions.skipBower) {
+          return bowerInstall.run({
+            verbose: commandOptions.verbose
```

Comment:
> any ember command will fail if bowerInstall did not run before

this is what I meant. I assume that you meant that it will fail because the bower deps are not installed, right? but the cache file doesn't actually depend on any bower dependencies afaik so we could just keep the npm first, bower second order if we manage to get around that "bower deps are not installed" check for this particular command.

the reason why I think this would be beneficial is that the `ember-cli` update instructions mention `npm install && bower install`, which would also fail if we keep the current solution.